.travis.yml

```diff
@@ -90,15 +90,10 @@ jobs:
           - gcc
       script:
         # Do a manual build+test sequence rather than using all.sh, because
-        # there's no all.sh component that does what we want. We should set
-        # CFLAGS for arm64 host CC.
+        # there's no all.sh component that does what we want.
```

Comment:
Should we clearly state what it is that we want? From what I gather that is to build project without the arm v8m-a architecture cryptographic [extension](https://developer.arm.com/documentation/100067/0612/armclang-Command-line-Options/-mcpu?lang=en)?

Reply:
For this PR, it is due to the travis time limitation. Before this PR, cpu modifiers is a MUST command line options.
On Arm64 host of Travis, test component will exceed the limitation.